src/inference/Inference.jl

```diff
@@ -300,7 +342,12 @@ function flatten_namedtuple(nt::NamedTuple)
     return [vn[1] for vn in names_vals], [vn[2] for vn in names_vals]
 end
 
-function get_transition_extras(ts::Vector)
+function get_transition_extras(ts::AbstractVector{<:VarInfo})
+    valmat = reshape([getlogp(t) for t in ts], :, 1)
+    return ["lp"], valmat
```

Comment:
```suggestion
    return ["logp"], valmat
```

Reply:
I called it "lp" since then it would be consistent with all other inference algorithms (`Transition`, `ParticleTransition`, and `HamiltonianTransition` all report "lp"). Maybe we should rename all to "logp" (but probably in a separate PR) or just keep "lp" here?

Reply:
Sounds good.